digitalpy/core/persistence/impl/default_persistence_facade.py

```diff
@@ -8,14 +8,14 @@
 #
 #######################################################
 from typing import Any
-from digitalpy.core.event_manager import EventManager
-from digitalpy.model.object_id import ObjectId
-from digitalpy.model.persistence_facade import PersistenceFacade
-from digitalpy.model.persistent_object import PersistentObject
-from digitalpy.model.build_depth import BuildDepth
-from digitalpy.model.paging_info import PagingInfo
-from digitalpy.model.persistence_mapper import PersistenceMapper
-from digitalpy.core.state_change_event import StateChangeEvent
+from digitalpy.core.util.event_manager import EventManager
+from digitalpy.core.domain.object_id import ObjectId
```

Comment:
**[pep8]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>I003 isort expected 1 blank line in imports, found 0